frontend/src/components/Sidepanel.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+    <VNavigationDrawer v-model="drawer" app clipped>
```

Comment:
You probably no need define v-model here and drawer prop, since you made drawer always opened, probably it's enough to use just `permanent` flag here

Reply:
should i consider the possibility of it being viewed on a mobile device and the drawer being closed by default?

Reply:
you can, but it will require different approach how to handle open/close action for it, since open action (button) could be placed in another component, like header, so if you want to do this, you need to emit event from Header component, whenever menu button clicked (it can use custom v-model to inform it's parent), and this component will pass drawer prop to v-navigation component, also it should work in reverse direction

so if you want to do so - you can try, but it probably not required for this ticket